app/views/setup/welcome.html.slim

```diff
@@ -9,7 +9,10 @@ h1 Initial CaaSP Configuration
 = form_for :settings, url: setup_path, method: :put do |f|
   .form-group
     = f.label :dashboard, "Dashboard location"
-    = f.text_field :dashboard, value: (Pillar.value(pillar: :dashboard) || request.host), class: "form-control"
+    = f.text_field :dashboard, value: (Pillar.value(pillar: :dashboard) || request.host), class: "form-control", required: true
+  .form-group
+    = f.label :apiserver, "External Kubernetes API server hostname"
```

Comment:
FQDN is likely better than hostname, I know, I know - this name was my suggestion, my bad! A hostname is `foo`, a FQDN is `foo.mycompany,com`.